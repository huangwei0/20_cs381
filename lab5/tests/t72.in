findall(Z, ancestor('Queen Elizabeth II', Z), Y), 
print(Y), 
print('\n'), 
halt.
