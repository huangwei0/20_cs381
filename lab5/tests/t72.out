[Edward: Earl of Wessex,Charles: Prince of Wales,Anne: Princess Royal,Andrew: Duke of York,Lady Louise Windsor,James: Viscount Severn,Prince William: Duke of Cambridge,Prince Henry of Wales,Prince George of Cambridge,Princess Charlotte of Cambridge,Peter Phillips,Zara Phillips,Savannah Phillips,Isla Phillips,Mia Tindall,Princess Beatrice of York,Princess Eugenie of York]
